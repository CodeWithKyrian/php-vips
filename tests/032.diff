001+ imageize: FIXME must also set at least interpretation