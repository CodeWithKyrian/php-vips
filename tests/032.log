
---- EXPECTED OUTPUT
pass
---- ACTUAL OUTPUT
imageize: FIXME must also set at least interpretation
pass
---- FAILED
