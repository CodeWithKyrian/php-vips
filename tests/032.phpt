--TEST--
__set and __get work
--SKIPIF--
<?php if (!extension_loaded("vips")) print "skip"; ?>
--FILE--
<?php 
  include 'vips.php';

  $filename = dirname(__FILE__) . "/images/img_0076.jpg";
  $image = Vips\Image::new_from_file($filename);

  $image->poop = "banana";
  $value = $image->poop;

  if ($value == "banana") {
	echo "pass";
  }
?>
--EXPECT--
pass
