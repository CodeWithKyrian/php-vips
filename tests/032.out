imageize: FIXME must also set at least interpretation
pass